jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -557,7 +557,7 @@ int jbmc_parse_optionst::doit()
   if(get_goto_program_ret != -1)
     return get_goto_program_ret;
 
-  if(!cmdline.isset("symex-driven-lazy-loading"))
+  if(!options.get_bool_option("symex-driven-lazy-loading"))
```

Comment:
Is there anything we could do to completely hide `cmdline` from such code so that it cannot wrongly be reintroduced?

Reply:
Will come with progress on https://github.com/diffblue/cbmc/pull/4404